### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by dberliner**
This contains two unrelated changes. I've revised the documentation and removed `xaprc-eval-tool` from default builds. Unless I'm mistaken the entire project was made obsolete by `xaprc-control` so complete removal may be a good idea.

### CODE REVIEW

**Review by drbild — CHANGES_REQUESTED** [review]
Just the one  questions about removing the documentation for building in multiple directories.  I think we want to keep that, but perhaps revised to fix the out-of-date defconfig names.

File: README.md [hunk 1]
```diff
@@ -31,40 +34,16 @@ make menuconfig
...
+
+# Or build the firmware locally. See buildroot docs for dependencies
+make BR2_JLEVEL=$(nproc)
 ```
 
-The firmware image will be built in
+The firmware image will be built as
 `images/emmc.img`.
 
-Use separate build directories to simplify working with multiple
```
[the remark this answers is not in the document]
Comment (dberliner) [reply README.md:41]: Yeah, I'll add that back.

File: README.md [hunk 2]
```diff
@@ -31,40 +34,21 @@ make menuconfig
...
 rebuild.
 
-``` bash
-cd build
-
-# Initialize a build directory for one config
-mkdir sama5d2_xplained_xaprc
-pushd sama5d2_xplained_xaprc
-
-make O=$PWD -C ../.. sama5d2_xplained_xaprc_defconfig
```
Comment (drbild) [inline README.md:53]: Can you keep the example code, up to this line?  I refer back to it for the correct syntax for initializing the defconfig from a separate build directory.  (The `make -O=$PWD -C ../.. <defconfig>` syntax).

**Review by drbild — CHANGES_REQUESTED** [review]

**Review by drbild — APPROVED** [review]
Once that `make -O=$PWD -C ../.. <defconfig>` line is added back, LGTM.

**Pull request merged**